client/app/index.jsx

```diff
@@ -33,7 +33,6 @@ class App extends React.Component {
       type: 'GET',
```

Comment:
Github won't let me comment up there, but you might want to refactor lines 12-23 to use Array constructor + Array.prototype.fill(), e.g. ```(new Array (12).fill({ ... });```